etc/gensbom/gensbom.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euo pipefail
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+
+WORKSPACE=${WORKSPACE:-${HERE}/.gensbom}
```

Comment:
I would just use `$PWD` instead of trying to figure out where the script is. You might also `echo creating SBOMs beneath $WORKSPACE` somewhere, too.
```suggestion
WORKSPACE=${WORKSPACE:-${PWD}/.gensbom}
```